### PR TITLE
Added setup instructions for OSX.

### PR DESCRIPTION
**Pull request by n1k0**
This one adds a new installation section to the docs, with specific setup for OSX. Another section for Linux should be added in a next PR.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Installing daybed
+=================
+
+Daybed has the following requirements:
+
+- A `Python <http://python.org/>`_ 2.6, 2.7, 3.3 or 3.4 installation
```
Comment (almet) [inline docs/source/install.rst]: Also, PyPy.

File: docs/source/install.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    $ make serve
+
+Development installation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you start hacking on daybed, a good practice is to ensure the tests keep
+passing for all supported Python environments::
+
+    $ brew install python python3 pypy couchdb
+    $ make install-dev
```
Comment (almet) [inline docs/source/install.rst]: install-dev is a dependency of "test", so you don't need this line.

File: docs/source/install.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Then, running the test suite is a good way to check that everything is going
+well, and is well installed. You can run them with::
+
+    $ make tests
+
+The test suite will run all the available tests for every supported Python
+environment. You can check the current build status
+`on Travis <https://travis-ci.org/spiral-project/daybed>`_.
+
+Mac OSX
```
Comment (almet) [inline docs/source/install.rst]: It seems that the installation instructions are pretty similar on debian and macos, maybe we should just put the two next to each other here? (I'm always reluctant to have macos only installation docs)

**Comment by almet** [comment]
Added a couple of comments. Sounds good overall.

File: docs/source/install.rst (hunk 1 above)
Comment (Natim) [inline docs/source/install.rst]: I also removed py33 from the tox test matrice. I think we can keep only the last Python3 version for now.

Comment (almet) [inline docs/source/install.rst]: +1

**Comment by n1k0** [comment]
Addressed comments.

**Pull request merged**